Mirror Bot Machine Broke (again)

MaltVinegar opened the pull request:
Mirrors
 BeeStation/BeeStation-Hornet#7476
 BeeStation/BeeStation-Hornet#7572
 BeeStation/BeeStation-Hornet#7709

## Changelog
:cl:
add: Medieval Mutation that modifies speech when applied
fix: cuffs are now properly accounted for in situations that affects all equipped items on a mob. This most notable affects when a mob transforms, where cuffs will now drop with everything else rather than being deleted.
tweak: Midround xenos now have a lower weight and will occur less often.
tweak: At 20 to 30 threat, midround xenos require two security instead of one.
/:cl:

Terra1 reviewed (approved):
Discord said it was okay.

itsmeow committed:
Lower xeno weight and require 2 sec at 20-30 threat range (#7709)

12Oreos committed:
Adds New Medieval Mutation (#7476)

* Update DNA.dm

* Create medieval_speech.json

* Update speech.dm

* Update speech.dm

* Update medieval_speech.json

* Update dna_injector.dm

* Update configuration.dm

* Update speech.dm

* Update speech.dm

Minor capitalization fix

* Update medieval_speech.json

More capitalization fixes and minor edits to some replacements

* Update speech.dm

Added the period

The-Moon-Itself committed:
Cuffs are now included in get_equipped_items() (mob transformation fix) (#7572)

KubeRoot commented:
Note - the austation mirrorbot has a `domirror` command that can be used to make it mirror PRs by ID - did you try that? I'm asking because sometimes it fails on very specific PRs, but otherwise this might not be necessary.

MaltVinegar commented:
I did not realize that
I can try it real quick
